### ISSUE TITLE
You never close the findAllShareFriends function

### ISSUE THREAD

**Issue by jewpaltz** [gorlagup1/fitness-tracker#2]
https://github.com/gorlagup1/fitness-tracker/blob/72b6cb5145c4a92ae567542c9b925cbc3aa1bc83/server/controllers/SharingWithFriends.js#L21

**Comment by gorlagup1** [comment]
Yeah actually I thought to add that that’s the reason I just created a view of that but then I left it and I added exercise page where different friends and users post and commit when the user login to the page .
